### PR TITLE
(SIMP-1445) Ensure packer can test ISO

### PR DESCRIPTION
**Pull request by jeannegreulich**
This sets up the simp-testing directory which has initial
  tests for building simp from the ISO and running initial tests.
  Still need to work with someone to integrate it with Travis.
  The test can be run individually.  See README in  simp-testing.

SIMP-1445 #comment needs integration with travis

---
This change is [Reviewable](https://reviewable.io/reviews/simp/simp-packer/5)

### CODE REVIEW

File: simp-testing/manifests/simpsetup/manifests/dns.pp [hunk 1]
```diff
@@ -0,0 +1,70 @@
+class simpsetup::dns
```
Comment (jeefberkey) [inline simp-testing/manifests/simpsetup/manifests/dns.pp]: You should consider bringing in our rake helpers so you can run `rake validate` and `rake lint`

File: simp-testing/README.md [hunk 2]
```diff
@@ -12,73 +12,178 @@
...
 
 ### Setup
 
 Requirements:
   - [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+  - [Vagrant](https://www.vagrantup.com/downloads.html)
   - SIMP ISO created from build:auto and its json file.
+    These contain the json variables that point to the iso,
+    the iso checksum and the name of the distribution iso.
+  - packer installed in or linked to /bin/packer. (There is
```
Comment (nick-markowski) [inline simp-testing/README.md]: Packer is distributed as a binary blob, we should just grab it automatically from source unless users specify a path via environment variable, something like SIMP_PACKER_CMD.

File: simp-testing/README.md [hunk 3]
```diff
@@ -12,73 +12,178 @@
...
-the start up scripts replaces the variables in the template/scripts with the value and copies the hardcoded
-script to the working environment.  In the template the $ is dropped from in from of the variable. 
+
+You don't have to set anything in the packer.yaml file on CentOS 7.  For CentOS 6
+you need to change the NAT_INTERFACE and HOST_ONLY_INTERFACE to eth0 and eth1
+respectively.
 
 
+2\. Run the test: Run the script:
+     TMPDIR=<some directory with lots of space> simp-packer_test.sh <full path to test directory from #1>
```
Comment (nick-markowski) [inline simp-testing/README.md]: ./simp_packer_test.sh

File: simp-testing/README.md [hunk 4]
```diff
@@ -12,73 +12,178 @@
...
-run using the sudo command.  To get around this and have one place to declare variables (packer.profile)
-the start up scripts replaces the variables in the template/scripts with the value and copies the hardcoded
-script to the working environment.  In the template the $ is dropped from in from of the variable. 
+
+You don't have to set anything in the packer.yaml file on CentOS 7.  For CentOS 6
+you need to change the NAT_INTERFACE and HOST_ONLY_INTERFACE to eth0 and eth1
+respectively.
 
 
+2\. Run the test: Run the script:
```
Comment (nick-markowski) [inline simp-testing/README.md]: You need to copy simp-testing to whatever dir you want to run simp_packer_test.sh in

File: simp-testing/README.md [hunk 5]
```diff
@@ -12,73 +12,178 @@
...
   - [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+  - [Vagrant](https://www.vagrantup.com/downloads.html)
   - SIMP ISO created from build:auto and its json file.
+    These contain the json variables that point to the iso,
+    the iso checksum and the name of the distribution iso.
+  - packer installed in or linked to /bin/packer. (There is
+    another command called packer in /usr/sbin/packer so I
+    point specifically to /bin/packer.)
 
 ### Usage
```
Comment (nick-markowski) [inline simp-testing/README.md]: Grepping around I see a bunch of useful environment variables, we should catalog those here

File: simp-testing/README.md [hunk 6]
```diff
@@ -12,73 +12,178 @@
...
 #### Simple build
-1\. To define a test, create a test directory and include the following three files:
+1\. To define a test, create a test directory and include the following three files
+ (There is more information on these three files in the samples directory.):
       vars.json:  The json file from the SIMP ISO.  If you have moved the iso
```
Comment (nick-markowski) [inline simp-testing/README.md]: This should be  formatted as a list

**Comment by nick-markowski** [comment]
I have one more readme update inbound, don't merge

**Pull request merged**